### DISCUSSION TITLE
Does ProseCodeInline support syntax highlighing?

### DISCUSSION THREAD

**Discussion by ijkml** [nuxt/content#2316] [Q&A] [answered]
I've been searching, this issue #2116 and the docs seem to suggest it does, but somehow I can't get it to work.

```md
`const codeInline: string = 'highlighted code inline'`{lang="ts"}
```

If it really does work, please point me to a working example/config. I could also open an issue and whip up a repro if needed.

Many thanks!

**Comment by nobkd** [comment, marked as answer]
In theory this should work. 

But it looks like there's an issue with [`nuxt-modules/mdc`](https://github.com/nuxt-modules/mdc).

I'll try to find the cause of the problem.

---

Ok, made a PR: nuxt-modules/mdc#39

Seems like this was not correctly moved when the MDC parser was extracted to its own module.

**Reply by ijkml** [reply]
Thank you @nobkd! I'll mark this as answer and track https://github.com/nuxt-modules/mdc/pull/39. Thanks again.
